### ISSUE TITLE
Need a strategy for images that duplicate text links (book covers, featured sets, etc.)

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#235]
The problem: when a book cover is linked, the URL is the same as the linked title. This is not optimal for accessibility. It's made slightly worse when not all titles have linked covers, so the link duplication is inconsistent.

For a keyboard user, it means and extra tab to move past each title that has a book cover. For a screen reader, the links list is bloated. And the link text for each image is not unique - it's "Book cover" for all of them.

Ideal outcome: one link per title, whether a book cover is present or not; book cover still clickable.

Possible solutions: 
a) put the image inside the title link, set alt="", and use CSS to place the image (may be extra-tricky with responsiveness)
b) use a JS onClick event instead of a href to link the covers
c) remove the images from the tab order using tabindex="-1" (this does not remove them from the links list, but it's better than nothing) 
d) and if we do c, the link should have title="" text that repeats the book title

(Same issue applies to the featured sets on the home page, but they may be easier to deal with.)
